src/core/CMakeLists.txt

```diff
@@ -1000,6 +1009,8 @@ INCLUDE_DIRECTORIES(
   metadata
   pal
   processing
+  providers
```

Comment:
I see, sounds good.

So what's the reason for shipping memory (and gdal/ogr) in core? Don't get me wrong, I don't see any downsides, just wondering if you've got some incredibly cool added value with shipping in core which I don't see.